### PR TITLE
Automated update of libcalico-go to master

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ import:
 - package: k8s.io/client-go
   version: 82aa063804cf055e16e8911250f888bc216e8b61
 - package: github.com/projectcalico/typha
-  version: 2343517a2f18d4a47216ad93308e6a71ad7e9dc2
+  version: 3439c0869c74b6d803ef9c947b1bd166f75ac8a0
```
Comment (bcreane) [inline glide.yaml]: How does the script know the typha pin already? Or does this need to be rev'ed once typha gets merged?

Comment (heschlie) [reply glide.yaml]: Looks like it grabs it when it revs typha? https://github.com/tigera/process/blob/master/releases/update-libcalico.sh#L59-L88

Comment (caseydavenport) [reply glide.yaml]: It's a fortune teller.

Comment (bcreane) [reply glide.yaml]: 🔮

Comment (bcreane) [reply glide.yaml]: as long as the typha build succeeds, all is well.

**Comment by bcreane** [comment]
now that typha has merged, I've confirmed the versions. approved!

**Pull request merged**